### DISCUSSION TITLE
Play games isolated on that virtual screen?

### DISCUSSION THREAD

**Discussion by DocMAX** [VirtualDrivers/Virtual-Display-Driver#226] [Q&A] [answered]
Is it possible to isolate games into that "window"?
Something that gamescope on linux does.

**Comment by bud3699** [comment]
This isnt really something we do, we just create the display in windows, someone might have made a windows app for it, but if i were going to guess, its windows so i assume no one has

**Comment by swirle13** [comment, marked as answer]
@DocMAX what you'll want is MonitorSwapper: https://github.com/Nonary/MonitorSwapAutomation/releases/latest. @Nonary has a bunch of useful scripts, like AutoHDR, ResolutionMatcher, RTSSLimiter, and PlayNiteWatcher, that can all be automatically used/installed when you run any of his apps. The plugins can be toggled on/off in any of the project's settings.json files.

I was using ResolutionMatcher before I came across this tool, so I'm going to switch to only using MonitorSwapper as VDD has a custom virtual monitor created for me and thus, I don't need a script to automatically change the resolution of my primary display (which has been kind of a headache).
